dts/msm8916/msm8916-zhihe-ufi001c.dts

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	model = "ufi-001c 4G Modem Stick";
+	compatible = "zhihe,ufi-001c", "qcom,msm8916", "lk2nd,device";
+	qcom,msm-id = <0xce 0x00 0xf8 0x00 0xf9 0x00 0xfa 0x00>;
```

Comment:
```suggestion
	qcom,msm-id = <206 0>;
```

MSM8916 with 4G modem.

Reply:
I don't think that's a good name...
Take a look at this thread(Chinese): [thread](https://www.kancloud.cn/handsomehacker/openstick/2636505)
There are actually many (at least 4) similar devices, they use different GPIO channels for LEDs and EDL button. The name `msm8916-512mb-mtp.dts` is not sufficient to distinguish between them.

Reply:
You can add nodes. See how `acer-a1-724` and `alcatel-idol347` are put in the same dts, [msm8916-mtp-smb1360.dts](https://github.com/msm8916-mainline/lk2nd/blob/master/dts/msm8916/msm8916-mtp-smb1360.dts).